Scenes/World/World.tscn

```diff
@@ -59,12 +61,11 @@ layer = 62
 
 [node name="EventManager" parent="." instance=ExtResource( 12 )]
 
-[node name="Music" parent="." instance=ExtResource( 13 )]
-stream = ExtResource( 14 )
-fade_in_time = 3.0
-fade_out_time = 5.0
-replay_delay = 5.0
-
 [node name="GameOverSound" type="AudioStreamPlayer" parent="."]
 stream = ExtResource( 15 )
 volume_db = -5.0
+
+[node name="MusicManager" parent="." instance=ExtResource( 16 )]
+main_menu_music = ExtResource( 17 )
+tavern_music = ExtResource( 13 )
+gameplay_music = ExtResource( 14 )
```

Comment:
I think, better make MusicManager autoloaded singleton. The music now isn't playing in the main menu screen, what is a bug. And it should be accessible in other scenes as well.